koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -393,8 +393,7 @@ def _write_demultiplexed_logs(self, build_id, logs_dir):
                 platform_logs[platform].write((entry.line + '\n').encode('utf-8'))
                 platform_logs[platform].flush()
             except Exception as error:
-                msg = "Exception (%s) while writing build logs: %s" % (type(error),
-                                                                       error)
+                msg = "Exception (%s) while writing build logs: %s", type(error), error
```

Comment:
Explain this please

Reply:
Why do you pass tuple to exception?

Reply:
This was a mistake from my side and this has already fixed